README.md

```diff
@@ -29,8 +28,16 @@ The documentation can be found under the following links:
 - for the [stable release](https://mne.tools/mne-bids/)
 - for the [latest (development) version](https://mne.tools/mne-bids/dev/index.html)
 
+Getting Support
+---------------
+
+For any usage questions, please post to the
+[MNE Forum](https://mne.discourse.group). Be sure to add the `mne-bids` tag to
+your question.
```

Comment:
nice :+1:

Reply:
btw @sappelhoff you can subscribe to individual tags, e.g. to get notifications for posts tagged as `men-bids`, go to https://mne.discourse.group/tag/mne-bids and click the bell